src/components/ApartmentApplicantList/index.js

```diff
@@ -2,23 +2,28 @@ import React, { Component } from 'react';
 import 'date-fns';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
-import ListItemAvatar from '@material-ui/core/ListItemAvatar';
-import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
 import ListItemText from '@material-ui/core/ListItemText';
-import Avatar from '@material-ui/core/Avatar';
-import IconButton from '@material-ui/core/IconButton';
-import PersonIcon from '@material-ui/icons/Person';
-import RemoveCircleOutlineIcon from '@material-ui/icons/RemoveCircleOutline';
-import user from '../../services/user';
+import GordonPeopleSearch from '../Header/components/PeopleSearch';
+import ApplicantListItem from './components/ApplicantListItem';
 import './apartmentApplicantList.scss';
+import '../../views/PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
 
+// Create a list of applicants, displayed by name, username, and class standing.
 export default class ApplicantList extends Component {
   constructor(props) {
     super(props);
-    this.renderApplicant = this.renderApplicant.bind(this);
+    this.handleSelection = this.handleSelection.bind(this);
     this.handleRemove = this.handleRemove.bind(this);
   }
 
+  handleSelection = theChosenOne => {
+    // Make sure the chosen username was not null
+    if (theChosenOne && theChosenOne !== null) {
+      // Send the selected username to the parent component
```

Comment:
Can you just say `if (theChosenOne) {...` to check whether it is null or not? 
(Same question about `profile` on line 29)

Reply:
Yes... I think I had it originally check for something other than null when developing the method inside the subclass, but then that got changed and I guess I forgot that I could just remove that second part

Reply:
Ah, whoops!